src/main/java/net/testudobank/MvcController.java

```diff
@@ -207,6 +207,11 @@ private void updateAccountInfo(User user) {
     user.setTransferHist(transferHistoryOutput);
   }
 
+  // Calculates the overdraft fee amount to an account when it is overdrawn
+  public int applyInterestRateToPennyAmount(int pennyAmount){
+    return ((int)(pennyAmount * INTEREST_RATE));
```

Comment:
You can get rid of the outer parentheses. They make this harder to read than it needs to be.